frontend/app/src/components/AssetBalances.vue

```diff
@@ -26,9 +26,8 @@
     </template>
     <template #item.usdValue="{ item }">
       <amount-display
-        show-currency="symbol"
-        :fiat-currency="item.asset"
-        :amount="item.amount"
+        :fiat-currency="'USD'"
```

Comment:
why are you hard coding USD here?

Reply:
It is needed for the amount-display convert the value to selected currency (USD indicate what currency used for the amount passed, in this case it is always USD value)

I think this part was not checked, when we update the <amount-display /> component

Reply:
Dashboard updated too

Reply:
wait there are other part i need to fix too

Reply:
I've fix several wrong implementation. It should be correct now